apps/backend/supabase/schema.gen.ts

```diff
@@ -34,13 +34,340 @@ export interface Database {
   }
   public: {
     Tables: {
-      [_ in never]: never
+      assistant_threads: {
```

Comment:
All of this file is generated by a script. This is what is used in the GitHub action that was failing. Supabase updated their DB and the latest docker image that was being pulled and the script ran against was missing types. This came from my lacal machine with latest DB up